clang/tools/driver/cc1depscan_main.cpp

```diff
@@ -966,12 +931,6 @@ int cc1depscand_main(ArrayRef<const char *> Argv, const char *Argv0,
       {
         ++NumRunning;
 
-        // In test mode, just tear down everything.
-        if (ShutDownTest) {
```

Comment:
Just FYI, this code was there to debug the situation that:
* Retry connection
* Daemon timeout mid build and you need to spawn a new daemon.

It is an interesting test case but I could not write a reliable test case to run. I am ok to remove it.